src/material-examples/expansion-steps/expansion-steps-example.css

```diff
@@ -0,0 +1,12 @@
+.headers-align .mat-expansion-panel-header-title,
+.headers-align .mat-expansion-panel-header-description {
+  flex-basis: 0;
+}
+
+.icons-to-right {
+    display: inline-block;
+    width: 100%;
+    text-align: right;
+    margin-top: 5px;
+    color: blue;
```

Comment:
@julianobrasil I think you're missing an eof newline here (see Travis).

Also, I'm not 100% sure, but I think all classes under material-examples folder, should be prefixed with `example-`.